File: labs-training-master/java/solid/solid-java-1/src/main/java/es/emergya/labs/training/solid_java_1/dip/excercise_00/Lamp.java
package es.emergya.labs.training.solid_java_1.dip.excercise_00;

public class Lamp implements Device {

	private int color;

	public void turnOn() {

		System.out.println("Lamp turned on");
	}

	public void turnOff() {

		System.out.println("Lamp turned off");
	}

	public int getColor() {

		return color;
	}

	public void setColor(int color) {

		this.color = color;
	}
}

File: labs-training-master/SolidPythonResuelto/main/dip/Lamp.py
from Device import Device

class Lamp(Device) :
    def turnOn(self) -> None:
        print("Lamp turned on")
    def turnOff(self) -> None:
        print("Lamp turned off")
    def  getColor(self) -> None:
        return  self.color
    def setColor(self, color:int) -> None:
        self.color:int = color